packages/setup-remote-server/lib/commands/server.js

```diff
@@ -0,0 +1,475 @@
+`;
+
+		const mediaRedirect = `  # Load media files from production server if they don't exist on staging.
+  RewriteCond %{REQUEST_FILENAME} !-f
+  RewriteRule ^(content/uploads/.*) ${ dotenvConfig.URL_STAGING }/$1 [L]
+
+  RewriteRule ^index\.php$ - [L]`;
+
+		htaccessContents = xRobotsTag + '\n' + basicAuth + '\n' + htaccessContents;
+		htaccessContents = htaccessContents.replace(
+			'  RewriteRule ^index\\.php$ - [L]',
+			mediaRedirect
+		);
+	}
+
+	try {
+		fs.writeFileSync( `${ dotLocalServer }/.htaccess.${ envoirnment }`, htaccessContents );
+		log( format.success( `.htaccess.${ envoirnment } saved to /.local-server` ) );
+	} catch ( err ) {
+		log( err );
+	}
+
+	// Sync files to remote server.
+	await ssh
+		.connect( {
+			host: hostName,
+			username: hostUser,
+			privateKey: untildify( privateKey ),
+		} )
+		.then( async () => {
+			// Make diretorires.
+			await ssh
+				.execCommand( `mkdir -p ${ remotePath }/shared/wordpress/content/uploads`, {
+					cwd: 'public_html',
+				} )
+				.then(
+					function () {
+						log( format.success( 'Directories create on remote server sucessfully' ) );
+					},
+					function ( error ) {
+						log( error );
+					}
+				);
+			// .env file.
+			await ssh
+				.putFile(
+					`${ dotLocalServer }/.env.${ envoirnment }`,
+					`${ remotePath }/shared/wordpress/.env`
+				)
+				.then(
+					function () {
+						log( format.success( '.env copied to remote server sucessfully' ) );
+					},
+					function ( error ) {
+						log( error );
+					}
+				);
+			// .htaccess.
+			await ssh
+				.putFile(
+					`${ dotLocalServer }/.htaccess.${ envoirnment }`,
+					`${ remotePath }/shared/wordpress/.htaccess`
+				)
+				.then(
+					function () {
+						log( format.success( '.htaccess copied to remote server sucessfully' ) );
+					},
+					function ( error ) {
+						log( error );
+					}
+				);
+			if ( envoirnment !== 'production' ) {
+				// .htpasswd.
+				await ssh
+					.putFile( `${ dotLocalServer }/.htpasswd`, `${ remotePath }/shared/.htpasswd` )
+					.then(
+						function () {
+							log(
+								format.success( '.htpasswd copied to remote server sucessfully' )
+							);
+						},
+						function ( error ) {
+							log( error );
+						}
+					);
+			}
+		} );
+
+	// Cleanup local files.
+	try {
+		fs.unlinkSync( `${ dotLocalServer }/.env.${ envoirnment }` );
+		fs.unlinkSync( `${ dotLocalServer }/.htaccess.${ envoirnment }` );
+		fs.unlinkSync( `${ dotLocalServer }/.htpasswd` );
+	} catch ( err ) {
+		log( err );
+	}
+
+	// Add comment to run deployment.
+
+	log( format.success( '\n✅  Done!' ) );
+	log(
+		`${ deployYMLData[ '.base' ].application } ${ envoirnment } is now installed under ${ remotePath }`
+	);
+	const repoWorkflowURl = deployYMLData[ '.base' ].repository
+		.replace( 'git@github.com:', 'https://github.com/' )
+		.replace( '.git', '/actions/workflows/deploy.yml' );
+	log( `Push a commit to GitHub or manually trigger a deployment: ${ repoWorkflowURl }` );
```

Comment:
Maybe recommend to use [`gh workflow run`](https://cli.github.com/manual/gh_workflow_run) instead.